deploy/sample-backup-configs/sample_config_build.ini

```diff
@@ -12,56 +12,37 @@ postbuildhook=
 [builds]
 # this section references builds defined in config_build_recipes.ini
 # if you add a build here, it will be built when you run buildafi
-# Legacy recipe without an L2
 firesim-quadcore-nic-llc4mb-ddr3
-#firesim-singlecore-no-nic-l2-lbp
-#firesim-singlecore-nic-l2-lbp
-#firesim-quadcore-no-nic-l2-lbp
-#firesim-quadcore-nic-l2-lbp
-firesim-quadcore-no-nic-l2-llc4mb-ddr3
-firesim-quadcore-nic-l2-llc4mb-ddr3
-#fireboom-singlecore-no-nic-l2-lbp
-fireboom-singlecore-no-nic-l2-llc4mb-ddr3
-#fireboom-singlecore-nic-l2-lbp
-fireboom-singlecore-nic-l2-llc4mb-ddr3
-#firesim-supernode-singlecore-nic-llc4mb-ddr3
-#firesim-supernode-quadcore-nic-llc4mb-ddr3
+firesim-quadcore-no-nic-llc4mb-ddr3
+fireboom-singlecore-no-nic-llc4mb-ddr3
+fireboom-singlecore-nic-llc4mb-ddr3
+
 firesim-supernode-singlecore-nic-lbp
 fireboom-singlecore-no-nic-l2-llc4mb-ddr3-ramopts
-# for MICRO 2019 tutorial. May be removed in the future.
-#firesim-singlecore-sha3-nic-l2-llc4mb-ddr3
-#firesim-singlecore-sha3-no-nic-l2-llc4mb-ddr3
-#firesim-singlecore-sha3-nic-l2-llc4mb-ddr3-print
-#firesim-singlecore-sha3-no-nic-l2-llc4mb-ddr3-print
-#fireboom-dualcore-no-nic-l2-llc4mb-ddr3
-#fireboom-dualcore-nic-l2-llc4mb-ddr3
-#firerocketboom-1rx1b-no-nic-l2-ddr3-llc4mb
+
+firesim-singlecore-gemmini-no-nic-llc4mb-ddr3
+
+# SHA3 configs for tutorial
+firesim-singlecore-sha3-no-nic-llc4mb-ddr3
```

Comment:
All remaining configs that were previously commented out should remain commented out..

Reply:
Do you mean to comment out these two new SHA3 configs? Or to add back the old `sha3-nic` to the new sample configs?

Reply:
The two sha3 configs and the gemini config should not be built by default. I'm alright with leaving them in config_build.ini but they should be commented out.

Reply:
And by built by default i mean, built if you type `firesim buildafi` without modifying the build.ini.

Reply:
The uncommented configs are the ones which have publicly available agfis, right?

@alonamid requested the gemmini config, and I agree with him it would be very useful for internal users, for the same reason why we have fireboom.

I'll comment the sha3 configs for now, but my motivation was that I would like the ASPLOS tutorial to be "checked in" to trunk branches of both chipyard and firesim. I want to avoid having to regenerate AFIs for all relevant tutorial configs before each tutorial.

Reply:
The context here is that we always have a problem with AGFI creep. We have FireBoom because has a considerable number of outside users, and long history of use in general. Gemmin[n]i is not there yet IMHO. My preference is that is for the gemmini developers to manage their own HWDB. 

I think the sha3 request is more reasonable. 

@sagark can tie break on this one.

Reply:
@jerryz123 how long do gemmini builds take? is there substantial overhead?

Is there a sha3 config in chipyard CI? If so, I don't think we get anything more out of shipping a sha3 config AGFI by default, unless someone is actually going to test it (which we don't currently have infrastructure for).

Reply:
The build I did this friday finished in 4 hours (judging by log timestamps and AGFI_INFO).

SHA3 config built-by-default reduces burden on tutorial preparation.

Reply:
@jerryz123 seems fine to me to keep one gemmini build then, assuming there are users interested

Re: SHA3
* I think it makes sense to add it to chipyard CI, to make sure it still works (sha3 in software sim running some simple test).
* Since we don't test generated AGFIs right now, I don't think it's useful to force these to be generated all the time. Building the AGFIs is not that much of a burden (assuming the build succeeds) and I don't think it's likely that chipyard CI on sha3 will pass, but an AGFI build will fail.

Reply:
Ok. SHA3 CI will not be included in this set of changes.

For now I'll leave the sha3 configs commented out in sample_configs, matching what existed before.